src/main/java/artifacts/item/trinket/TrinketArtifactItem.java

```diff
@@ -192,32 +218,4 @@ private static String getEffectsEnabledLanguageKey(ItemStack stack) {
 		return effectsEnabled(stack) ? "artifacts.trinket.effectsenabled" : "artifacts.trinket.effectsdisabled";
 	}
 
-	// From Curios
```

Comment:
You can't remove this as it serves a real purpose. Some artifacts have sounds with a different pitch an volume.